lms/static/js/groups/views/cohort_discussions.js

```diff
@@ -1,99 +1,98 @@
-var edx = edx || {};
-
-(function ($, _, Backbone, gettext, interpolate_text, NotificationModel, NotificationView) {
+;(function (define) {
     'use strict';
+    define(['jquery', 'underscore', 'backbone', 'gettext', 'js/models/notification', 'js/views/notification'], 
```

Comment:
If interpolate_text is used in this file, declare the dependency on string_utils, which is shimmed to map to interpolate_text:

"string_utils": {
                deps: ["underscore"],
                exports: "interpolate_text"
            },